modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,612 @@
++
+[source,terminal]
+----
+$ oc delete node openshift-control-plane-2
+
+node "openshift-control-plane-2" deleted
+----
++
+Verify the node has been deleted:
++
+[source,terminal]
+----
+$ oc get nodes
+
+NAME                     STATUS ROLES   AGE   VERSION
+openshift-control-plane-0 Ready master 3h24m v1.24.0+9546431
+openshift-control-plane-1 Ready master 3h24m v1.24.0+9546431
+openshift-compute-0       Ready worker 176m v1.24.0+9546431
+openshift-compute-1       Ready worker 176m v1.24.0+9546431
+----
+
+. Create the new `BareMetalHost` object and the secret to store the BMC credentials:
+
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-control-plane-2-bmc-secret
+  namespace: openshift-machine-api
+data:
+  password: <password>
+  username: <username>
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-control-plane-2
+  namespace: openshift-machine-api
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: redfish://10.46.61.18:443/redfish/v1/Systems/1
+    credentialsName: openshift-control-plane-2-bmc-secret
+    disableCertificateVerification: true
+  bootMACAddress: 48:df:37:b0:8a:a0
+  bootMode: UEFI
+  externallyProvisioned: false
+  online: true
+  rootDeviceHints:
+    deviceName: /dev/sda
+  userData:
+    name: master-user-data-managed
+    namespace: openshift-machine-api
+EOF
+----
++
+Note: username and password can be found from the other bmh's secrets. The protocol to use in bmc:address: can be taken from other bmh objects.
++
+[IMPORTANT]
+====
+If you reuse the `BareMetalHost` object definition from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
+====
++
+After the inspection is complete, the `BareMetalHost` object is created and available to be provisioned.
+
+. Verify the creation process using available `BareMetalHost` objects:
++
+[source,terminal]
+
+----
+$ oc get bmh -n openshift-machine-api
+
+NAME                      STATE                  CONSUMER                      ONLINE ERROR   AGE
+openshift-control-plane-0 externally provisioned examplecluster-control-plane-0 true         4h48m
+openshift-control-plane-1 externally provisioned examplecluster-control-plane-1 true         4h48m
+openshift-control-plane-2 available              examplecluster-control-plane-3 true         47m
+openshift-compute-0       provisioned            examplecluster-compute-0       true         4h48m
+openshift-compute-1       provisioned            examplecluster-compute-1       true         4h48m
+----
++
+.. Create the new control plane machine using the `new-master-machine.yaml` file:
```

Comment:
These substeps seem disconnected from the parent step. Should they be their own steps?

Reply:
Hmmm. i'm pondering that. I wanted the a,b,c,d to be "components of the 9th step". But that might be a bit complex? I'm thinking on it.....